app/src/main/res/xml/pref_misc.xml

```diff
@@ -1,91 +1,92 @@
+            android:key="key_misc_rounded_corners"
+	          android:title="Set rounded corners diameter"
+	          android:inputType="number" />
+		<EditTextPreference
         android:defaultValue="-1"
         android:key="key_misc_rounded_corners_overlay"
         android:title="Set forced/faked rounded corners diameter"
 	android:inputType="number" />
-    <PreferenceCategory android:title="Backlight">
-        <CheckBoxPreference
+		<CheckBoxPreference
             android:defaultValue="false"
             android:key="key_misc_linear_brightness"
             android:title="Use linear screen brightness slider" />
-        <CheckBoxPreference
+		<CheckBoxPreference
             android:defaultValue="false"
             android:key="key_misc_disable_buttons_backlight"
             android:title="Disable backlight of hardware buttons if present (requires reboot)" />
-    </PreferenceCategory>
-    <ListPreference
+	</PreferenceCategory>
+	<PreferenceCategory android:title="Other">
+		<ListPreference
         android:defaultValue="-1"
         android:entries="@array/pref_misc_bluetooth"
         android:entryValues="@array/pref_misc_bluetooth_values"
         android:key="key_misc_bluetooth"
         android:title="Bluetooth workarounds" />
-    <Preference android:title="Securize"
+		<Preference android:title="Securize"
         android:key="key_misc_securize" />
-    <CheckBoxPreference
+		<CheckBoxPreference
         android:defaultValue="false"
         android:key="key_misc_remotectl"
         android:title="Allow takeover of the device by phh for debugging" />
+		<ListPreference
+    	    android:defaultValue="1"
+    	    android:entries="@array/pref_misc_fps_divisor"
+    	    android:entryValues="@array/pref_misc_fps_divisor_values"
+    	    android:key="key_misc_fps_divisor"
+    	    android:title="FPS Divisor" />
+		<ListPreference
+    	    android:defaultValue="-1"
+    	    android:entries="@array/pref_misc_display_fps"
+    	    android:entryValues="@array/pref_misc_display_fps_values"
+    	    android:key="key_misc_display_fps"
+    	    android:title="Force FPS" />
+		<ListPreference
+    	    android:defaultValue="1.86"
+    	    android:entries="@array/pref_misc_max_aspect_ratio_pre_o"
+    	    android:entryValues="@array/pref_misc_max_aspect_ratio_pre_o_values"
+    	    android:key="key_misc_max_aspect_ratio_pre_o"
+    	    android:title="Compatibility max aspect ratio" />
+		<CheckBoxPreference
+            android:defaultValue="false"
+            android:key="key_misc_multi_camera"
+            android:title="Expose Aux cameras (Qualcomm)" />
+		<CheckBoxPreference
+    		android:defaultValue="false"
+    		android:key="key_misc_force_camera2api_hal3"
+    		android:title="Force Enable Camera2API HAL3" />
+		<CheckBoxPreference
+            android:defaultValue="false"
+            android:key="key_misc_force_navbar_off"
+            android:title="Force navigation bar disabled" />
+		<CheckBoxPreference
```

Comment:
Could be moved with key_huawei_headset_fix as "Audio" section

Reply:
How is navbar in audio?

Reply:
Right ofc, Done